docs/concepts/tokens/decentralized-exchange/automated-market-makers.md

```diff
@@ -89,7 +95,7 @@ To prevent misuse, some restrictions apply to the assets used in an AMM. If you
 - If the asset is a token whose issuer uses [Authorized Trust Lines](../fungible-tokens/authorized-trust-lines.md), the creator of the AMM must be authorized to hold those tokens. Only your authorized trust lines can deposit that token into the AMM or withdraw it; however, you can still deposit or withdraw the other asset.
 - If the [Clawback amendment][] is enabled, the issuer of the token must not have enabled the ability to claw back their tokens.
 
-## LP Tokens
+## AMM and LP Tokens
```

Comment:
These "AMM and..." changes annoy me since it's on the AMM page, but I can see how they could help with SEO in some places. But they might break some links.

Reply:
Agree - while they are not ideal and feel redundant, these updates are intended to help with SEO.